crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -123,11 +123,11 @@ def repocheck
           } unless os_available
 
           cloud_available = repo_version_available?(products, "suse-openstack-cloud", "8")
-          ret[:cloud] = {
+          ret[:openstack] = {
             available: cloud_available,
             repos: {}
           }
-          ret[:cloud][:repos][admin_architecture.to_sym] = {
+          ret[:openstack][:repos][admin_architecture.to_sym] = {
             missing: ["SUSE OpenStack Cloud 8"]
```

Comment:
names should be fixed like in the backport: https://github.com/crowbar/crowbar-core/pull/724
```
"SUSE-OpenStack-Cloud-8-Pool",
"SUSE-OpenStack-Cloud-8-Updates"
```